README.md

```diff
@@ -156,8 +156,8 @@ To generate secure random IDs in React Native, you must use
 
 ```js
 const generateSecureRandom = require('react-native-securerandom').generateSecureRandom
-const format = require('nanoid/async/format')
-const url = require('nanoid/url')
+const format = require('nanoid/dist/cjs/async/format')
+const url = require('nanoid/dist/cjs/url')
```

Comment:
It is a breaking change. It can’t be done in 2.1 and you need a really good reason to make 3.0.

Reply:
yes of course

i think, if nanoid was ever to provide esm format, it would be best to abandon this pr, and take it from a better approach:

1. leave all the commonjs files in-place (don't organize them into a `source` dir)
2. compile esm to `dist-esm`

this way, esm could be a non-breaking addition, which would be much better